src/components/BlogPost/blogs.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import weekendfly from '../../images/weekendfly.jpg';
+import redclay from '../../images/redclay.jpg';
+import nightfamily from '../../images/nightfamily.jpg';
+
+const blogList = [
+  {
+    name: 'The Last Shadow Puppets – The Age of the Understatement',
+    blogSideDate: 'September 2, 2017',
+    blogSideTags:
+      '60s, Alex Turner, Arctic Monkeys, Bond, James Bond, rock, The Last Shadow Puppets',
+    description:
+      "Arctic Monkeys frontman Alex Turner teams up with The Rascals’ Miles Kane to make the perfect soundtrack to a Bond movie if it was ever a Western. Heavily influenced by equal parts ’60s English pop rock and nostalgic heartache, this vivid and indulgent project will have you swaying to lush string arrangements and yearning harmonies while Alex’s signature songwriting will leave you with “paper cuts from the love letters you never gave him.”"
+      ,
+    link: (
+      <iframe
+        src="https://open.spotify.com/embed/album/4fxflUQfRpsSPzVVBMZLQm"
```

Comment:
All these iframes are missing a unique title property. This iframe can also be self closing. For example
```<div></div>``` can be shortened to ```<div />```. Both these showed up in the terminal as compiled with warnings. try to resolve these warnings so you can have a clean compile and so debugging isnt too cluttered in the future.